Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -797,31 +824,50 @@ function Add-MetadataAndStructures() {
     )
     $roleAdded = Add-EdwAdminRole -RoleName "DataProcessingServiceUser" -Config $Config
 
-    $terminologyExists = Test-DataMartExists -Name "Terminology" -Config $Config
-    $sharedTerminologyExists = Test-DataMartExists -Name "SharedTerminology" -Config $Config
-
-    if (($terminologyExists -eq $false) -and ($sharedTerminologyExists -eq $false)) {
+    $terminologyDataMartId = Get-DataMartId -Name "Terminology" -Config $Config
+    if ($terminologyDataMartId -eq $null) {
         # POST Terminology data marts to MDS
         $terminologyDataMartId = Invoke-PostToMds -Config $Config -name "Terminology" -path ".\Terminology.json"
 
         if(!$terminologyDataMartId) {
             Write-DosMessage -ErrorAction Stop -Level "Error" -Message "The Terminology metadata could not be created. Please resolve the error with POSTing the Terminology data mart to the metadata service"#
         }
-        
+    } else {
+        Write-DosMessage -Level "Information" -Message "Terminology metadata already exists, skipping step"
+    }
+    
+    $sharedTerminologyDataMartId = Get-DataMartId -Name "SharedTerminology" -Config $Config
+    if ($sharedTerminologyDataMartId -eq $null) {
+        # POST Terminology data marts to MDS
         $sharedTerminologyDataMartId = Invoke-PostToMds -Config $Config -name "SharedTerminology" -path ".\SharedTerminology.json"
         if(!$sharedTerminologyDataMartId) {
             Write-DosMessage -ErrorAction Stop -Level "Error" -Message "The SharedTerminology metadata could not be created. Please resolve the error with POSTing the SharedTerminology data mart to the metadata service"
         }
-        
+    } else {
+        Write-DosMessage -Level "Information" -Message "Shared Terminology metadata already exists, skipping step"
+    }
+
+    Test-DatabaseExists -SqlAddress $Config.edwAddress -Name "Terminology"
+
+    $viewExists = Test-ViewExists -SqlAddress $Config.edwAddress -Database "Terminology" -Schema "Open" -View "HHSHCCOverrides"
```

Comment:
We're only checking two views, correct? HHSHCCOverrides and ValueSetDescription

Reply:
yes, they are the last things to get created in the load. So I figured it was pretty safe